<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StartWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnstart">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>开始教学</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnesc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnintroduce">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>功能介绍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnnew">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>新棋局</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnhelp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>作品说明</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnback">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>601</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncloseBrowser">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncloseBrowser_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
